examples/advanced/custom-filters/README.md

```diff
@@ -0,0 +1,61 @@
+## Customize token output using filters
+
+This example shows how you can manage what tokens are generated and how they are organized. This is useful when you want to generate a 1:1 relationship between files and token categories.
+
+Common use cases include:
+
+- Each token category as its own Sass partial (_colors.scss)
+- Separate component files (button.css, input.css, etc)
+- Tree shaking (only import what you need)
```

Comment:
I love this 'common use cases' section. I think this is something we should start adding to examples.